apps/api/src/services/baton-mcp.client.ts

```
@@ -0,0 +1,81 @@
+  ): Promise<McpToolResult> {
+    const transport = new StreamableHTTPClientTransport(new URL(this.resourceUrl), {
+      fetch: this.fetcher,
+      requestInit: { headers: { Authorization: `Bearer ${accessToken}` } },
+    });
+    const client = new Client({ name: 'convolens', version: '1.0.0' });
+    try {
+      await client.connect(transport);
+      return (await client.callTool(
+        { name, arguments: args },
+        undefined,
+        { timeout: BATON_TIMEOUT_MS }
```

ESLint

[error] Replace `⏎········{·name,·arguments:·args·},⏎········undefined,⏎········{·timeout:·BATON_TIMEOUT_MS·}` with `{·name,·arguments:·args·},·undefined,·{⏎········timeout:·BATON_TIMEOUT_MS,`

(prettier/prettier)